jbpm-test-coverage/src/test/java/org/jbpm/test/functional/ErrorLinkProcessTest.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * To change this template, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.jbpm.test.functional;
+
+import org.jbpm.test.JbpmTestCase;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.kie.api.runtime.KieSession;
+
+import static org.junit.Assert.assertNotNull;
+
+/**
+ */
+public class ErrorLinkProcessTest extends JbpmTestCase {
+
+    @Rule
+    public ExpectedException exceptionRule = ExpectedException.none();
+
+    public static final String PROCESS_EMPTY = "org/jbpm/test/functional/common/EmptyLinkProcess.bpmn2";
+    public static final String PROCESS_EMPTY_ID = "org.jbpm.test.functional.common.EmptyLinkProcess";
+    
+    public static final String PROCESS_MULTI_THROW = "org/jbpm/test/functional/common/MultipleThrowLinkProcess.bpmn2";
+    public static final String PROCESS_MULTI_THROW_ID = "org.jbpm.test.functional.common.MultipleThrowLinkProcess";
+    
+    public static final String PROCESS_MULTI_CATCH = "org/jbpm/test/functional/common/MultipleCatchLinkProcess.bpmn2";
+    public static final String PROCESS_MULTI_CATCH_ID = "org.jbpm.test.functional.common.MultipleCatchLinkProcess";
+    
+    public static final String PROCESS_UNCONNECTED = "org/jbpm/test/functional/common/UnconnectedLinkProcess.bpmn2";
+    public static final String PROCESS_UNCONNECTED_ID = "org.jbpm.test.functional.common.UnconnectedLinkProcess";
+
+    public ErrorLinkProcessTest() {
+        super(false);
+    }
+
+    @Test(timeout = 30000)
+    public void testEmptyLinkEvents() {
+        exceptionRule.expect(IllegalArgumentException.class);
+        exceptionRule.expectMessage("nodes do not have a name");
+        KieSession ksession = createKSession(PROCESS_EMPTY);
+        ksession.startProcess(PROCESS_EMPTY_ID);
```

Comment:
We don't really need to start any process for any of these tests as in case of any exceptions/errors these will be thrown while creating Kie session.